docs/troubleshooting.md

```diff
@@ -217,3 +217,91 @@ The following is a simple example of this occurring:
+| net_conn_deltas                                  | Number of connection events sent to Sensor.                                                                                          |
+| net_conn_inactive                                | Accumulated number of connections destroyed (closed)                                                                                 |
+| net_cep_updates                                  | Each time an endpoint object is updated in the model (scrapes only).                                                                 |
+| net_cep_deltas                                   | Number of endpoint events sent to Sensor.                                                                                            |
+| net_cep_inactive                                 | Accumulated number of endpoints destroyed (closed)                                                                                   |
+| net_known_ip_networks                            | Number of known-networks defined.                                                                                                    |
+| net_known_public_ips                             | Number of known public addresses defined.                                                                                            |
+| process_lineage_counts                           | Every time the lineage info of a process is created (signal emitted) \[1\]                                                             |
+| process_lineage_total                            | Total number of ancestors reported \[1\]                                                                                               |
+| process_lineage_sqr_total                        | Sum of squared number of ancestors reported \[1\]                                                                                      |
+| process_lineage_string_total                     | Accumulated size of the lineage process exec file paths \[1\]                                                                          |
+| process_info_hit                                 | Accessing originator process info of an endpoint with data readily available.                                                        |
+| process_info_miss                                | Accessing originator process info of an endpoint ends-up waiting for Falco to resolve data.                                          |
+| rate_limit_flushing_counts                       | Number of overflows in the rate limiter used to send process signals.                                                                |
+
+\[1\] the process lineage information contains the ancestors list of a process. This attribute is formatted as a list of
+the process exec file paths.
+
+### Sysdig stats -> rox_collector_events
```

Comment:
This notation with `->` is a bit confusing, is it important? Maybe simply `Sysdig stats: rox_collector_events`

Reply:
Actually, the left part is the component name the stats come from, and the right part is the name of the set of metrics in prometheus.

Reply:
So, what the arrow is representing?

Reply:
I did an attempt to use plain section titles and add a header with normalized format at the beginning. What do you think, @erthalion ?

Reply:
Looks good to me, thanks!